nx.json

```diff
@@ -46,5 +46,6 @@
   "nxCloudAccessToken": "MWM4ZWZiNDgtOGYwMi00ZDI3LTllMDUtYTNkYTExMmU3MDY2fHJlYWQ=",
   "parallel": 3,
   "defaultBase": "origin/main",
-  "useInferencePlugins": false
+  "useInferencePlugins": false,
+  "useLegacyCache": true
```

Comment:
This may come from the automatic migrations, but using the new DB cache shouldn't be an issue for this project right?